src/containers/Admin/EditTab/index.tsx

```diff
@@ -98,6 +103,21 @@ const EditTab = (): JSX.Element => {
         }
     }, [debouncedDistance, setSettings, settings])
 
+    useEffect(() => {
+        if (
+            !(showIcon || showTemperature || showWind || showPrecipitation) &&
```

Comment:
Her kan du bruke deMorgan for å sleppe parentes. Blir litt enklare å lese kanskje? `!showIcon && !showTemperature && !showWind && !showPrecipitation && showWeather`.

Reply:
Endret nå